homeassistant/const.py

```diff
@@ -934,6 +934,13 @@ class UnitOfSpeed(StrEnum):
 """Deprecated: please use UnitOfVolumetricFlux.INCHES_PER_HOUR"""
 
 
+# Wind_speed units
+class UnitOfWindSpeed(StrEnum):
+    """Wind speed units."""
+
+    BEAUFORT = "Bft"
```

Comment:
This has impact on our models. This needs to be proposed, discussed and approved in our architectural repository before creating a PR for it.

Please open a proposal there first.

../Frenck

Reply:
As you wish: https://github.com/home-assistant/architecture/discussions/1020

Reply:
Thanks!